src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/python/tests/test_organizations.py

```diff
@@ -314,5 +314,489 @@ def test_build_account_path(cls):
+                }
+            ]
+        }
+
+        expected_response = [
+            {
+                "Id": "11111111111",
+                "Arn": "",
+                "Email": "account+1@example.com",
+                "Status": "ACTIVE",
+                "JoinedMethod": "Invited",
+                "JoinedTimestamp": datetime(2022, 8, 9),
+            },
+            {
+                "Id": "22222222222",
+                "Arn": "",
+                "Email": "account+2@example.com",
+                "Status": "ACTIVE",
+                "JoinedMethod": "Invited",
+                "JoinedTimestamp": datetime(2022, 8, 9),
+            },
+            {
+                "Id": "444444444",
+                "Arn": "",
+                "Email": "account+4@example.com",
+                "Status": "ACTIVE",
+                "JoinedMethod": "Invited",
+                "JoinedTimestamp": datetime(2022, 9, 26),
+            },
+        ]
+
+        org_stubber.add_response("list_roots", list_roots_response)
+        org_stubber.add_response(
+            "list_organizational_units_for_parent",
+            list_organizational_units_for_root_response,
+            {"ParentId": "r-1337"},
+        )
+        org_stubber.add_response(
+            "list_organizational_units_for_parent",
+            list_organizational_units_for_production_response,
+            {"ParentId": "ou-123456"},
+        )
+        org_stubber.add_response(
+            "list_accounts_for_parent",
+            list_accounts_for_banking_response_page_0,
+            {"ParentId": "ou-080922"},
+        )
+        org_stubber.add_response(
+            "list_accounts_for_parent",
+            list_accounts_for_banking_response_page_1,
+            {"ParentId": "ou-080922", "NextToken": "PAGE1"},
+        )
+        org_stubber.add_response(
+            "list_organizational_units_for_parent",
+            list_organizational_units_for_banking_response,
+            {"ParentId": "ou-080922"},
+        )
+
+        org_stubber.add_response(
+            "list_accounts_for_parent",
+            list_accounts_for_commercial_response_page_0,
+            {"ParentId": "ou-26092022"},
+        )
+
+        org_stubber.add_response(
+            "list_organizational_units_for_parent",
+            list_organizational_units_for_commercial_response,
+            {"ParentId": "ou-26092022"},
+        )
+
+        org_stubber.activate()
+        tag_stubber.activate()
+        organizations = Organizations(
+            role=None, org_client=org_client, tagging_client=tagging_client
+        )
+        response = organizations.get_accounts_in_path(
+            "/production/banking",
+            resolve_children=True,
+            excluded_paths=["/production/banking/investment"],
+        )
+
+        self.assertListEqual(expected_response, list(response))
 
-    assert cls.build_account_path('some_ou_id', [], cache) == 'some_ou_name'
+class OrgClientInitTestCases(unittest.TestCase):
```

Comment:
There need to be two blank lines before new class definitions. Odd the linter didn't spot this.